### ISSUE TITLE
(func discretize) Number of splits does not respect bins parameter 

### ISSUE THREAD

**Issue by FeatCrush** [vertica/VerticaPy#156]
When using discretize func with bins parameter equal to X, the resulting column contains X -1 splits.

## Expected : 
total_values = 1000
bins = 5 
splits [1, 200, 400, 600, 800, 1000]

## Actual: 
total_values = 1000
bins = 5 
splits [1, 250, 500, 750, 1000]

[This behavior is found in TU's](https://github.com/vertica/VerticaPy/blob/a188213576fa71e5e1623d1cb46fd4f14cc8e41f/verticapy/tests/vDataFrame/test_vDF_preprocessing.py#L85)

**Comment by oualib** [comment]
Indeed, the behavior of the function is not corrected when parameter method is set to 'same_freq'. It is corrected and it will be available in v0.7.0.
